<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecoverWidget</class>
 <widget class="QWidget" name="RecoverWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>340</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: #2c3e50;
}
QLabel{
	font-family: Roboto;
	font-weight: 100;
	font-size: 34px;
	color: #ecf0f1;
}
QWidget#RecoverWidget QPushButton#btnLogin{
	font-size: 14px;
	background-color: #2c3e50;
}
QLineEdit{
	height: 32px;
	font-family: Roboto;
	font-size: 16px;
	color: #ecf0f1;
	background-color: #2c3e50;
	border: 0px;
	border-bottom: 1px solid white;
}
QPushButton{
	font-family: Roboto;
	font-size: 16px;
	font-weight: 300;
	color: #ecf0f1;
	border: 0;
	border-radius: 3px;
	background-color: #8e44ad;
}
QPushButton:hover{
	background-color: #9b59b6;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>320</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="_2">
        <item>
         <widget class="QLabel" name="lblTitle">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="text">
           <string>Recuperação</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtUsername">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maxLength">
           <number>16</number>
          </property>
          <property name="placeholderText">
           <string>Usuário</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtEmail">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maxLength">
           <number>255</number>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
          <property name="placeholderText">
           <string>Email</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRecover">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>RECUPERAR</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLogin">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Já sabe suas credenciais?</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblStatus">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtUsername</tabstop>
  <tabstop>txtEmail</tabstop>
  <tabstop>btnRecover</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
